### ISSUE TITLE
Set not display search status information on airline not works.

### ISSUE THREAD

**Issue by roachsinai** [osyo-manga/vim-anzu#29]
```
Plug 'osyo-manga/vim-anzu'
let g:airline#extensions#anzu#enabled = 0
```

Above setting still has anzu status in airline.

**Comment by roachsinai** [comment]
if I have below setting:

```
let g:anzu_airline_section = "x"
let g:airline#extensions#anzu#enabled = 1
```

I got this:

![image](https://user-images.githubusercontent.com/9500049/95874945-0b480900-0da4-11eb-96da-cefd5daeb51a.png)

There are two `anzu` status. If I set `let g:airline#extensions#anzu#enabled = 0` there will still have one.

**Comment by roachsinai** [comment]
I use vim-plug to manage vim plugins. Anzu still works even I remove it from

```
call plug#begin('~/.vim/plugged')
call plug#end()
```

`vim-anzu` not show in `&rtp`, but still show in airline.

It's wierd...

Thanks for any hints!

**Comment by osyo-manga** [comment]
hi.
Does setting `let g:anzu_enable_CursorHold_AnzuUpdateSearchStatus = 1` solve it?

**Comment by roachsinai** [comment]
Maybe not.

current setting is:

```
Plug 'osyo-manga/vim-anzu'
let g:anzu_enable_CursorHold_AnzuUpdateSearchStatus = 1
```

and return two search status:

![Screen Capture_select-area_20201013230602](https://user-images.githubusercontent.com/9500049/95879513-06398880-0da9-11eb-9ef6-d1ed4fdf6825.png)

**Comment by roachsinai** [comment]
So their should be two `joplin[6/7]` here? one surround by `[]` one by `()`?

**Comment by osyo-manga** [comment]
How does it work when setting ?

```vim
let g:airline#extensions#anzu#enabled = 0
let g:anzu_enable_CursorHold_AnzuUpdateSearchStatus = 1
```

**Comment by roachsinai** [comment]
Setting above leaving only `joplin[6/7]`. Is there a choice to hide even the last `joplin[6/7]`? As combine with https://github.com/qxxxb/vim-searchhi will show this at the very bottom.

![Screen Capture_select-area_20201013234051](https://user-images.githubusercontent.com/9500049/95883776-e2c50c80-0dad-11eb-88c3-d8546384e2c0.png)

**Comment by roachsinai** [comment]
What's more seems `let g:anzu_airline_section = "C"` not works, still show in section `Y`.

**Comment by osyo-manga** [comment]
> So sorry for so much questions...

It's not a problem. Thank you.
However, my reply is slow ... I'm sorry.

**Comment by roachsinai** [comment]
So kind of you, wish there is a space to split contents in section `y`.

**Comment by roachsinai** [comment]
https://github.com/osyo-manga/vim-anzu/blob/1b95ec384487e559f0ee70e0f46c33683ff533ac/autoload/airline/extensions/anzu.vim#L12

I found in code above has a space in `" %{anzu#search_status()}"`, but not works, even I add one more space.

**Comment by roachsinai** [comment]
Closed by same reason as #30 .
